alpha_challenges/the_basics.md

```diff
@@ -143,8 +148,9 @@ end
 **Answer 11:**
 ```ruby
 def names
-  # write your code in here
+  puts ["Jackie", "Hami"]
```

Comment:
In Ruby, it's usually best practice to use single quotes instead of double quotes. Reserve double quotes for string interpolation.

Reply:
I'll keep that in mind, thank you!